addons/mail/static/src/messaging/component/chatter_topbar/chatter_topbar.scss

```diff
@@ -26,6 +33,15 @@
     border-left: none;
 }
 
+.o_ChatterTopbar_buttonClose {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 34px;
+    min-width: 34px;
```

Comment:
Is this a flex specific thing? `min-width` seems unnecessary if `width` has the same value?

Reply:
I don't really know why but when flexbox is shrink, it can ignore the width